src/hupper/reloader.py

```diff
@@ -133,21 +133,27 @@ def run(self):
         """
         Execute the reloader forever, blocking the current thread.
 
-        This will invoke ``sys.exit(1)`` if interrupted.
+        This will invoke ``sys.exit(1)`` if interrupted and
+        the process didn't handle the interrupt gracefully.
 
         """
+        exitcode = 1
```

Comment:
we should probably switch this up and go with -1 instead and try to reserve the primary codes for the user's worker